internal/commands/fastedge/logs.go

```diff
@@ -18,11 +18,11 @@ import (
 )
 
 func appLogsFilterFlags(cmd *cobra.Command) {
-	cmd.Flags().StringP("from", "", "", "From time")
-	cmd.Flags().StringP("to", "", "", "To time")
-	cmd.Flags().StringP("sort", "", "asc", "Sort order")
-	cmd.Flags().StringP("edge", "", "", "Edge name")
-	cmd.Flags().StringP("client-ip", "", "", "Client IP")
+	cmd.Flags().String("from", "today", "Reporting period start, UTC")
+	cmd.Flags().String("to", "now", "Reporting period end, UTC")
+	cmd.Flags().String("sort", "asc", "Log sort order, asc or desc")
+	cmd.Flags().String("edge", "", "Edge name filter")
+	cmd.Flags().String("client-ip", "", "Client IP filter")
```

Comment:
Please hide this flag for now using https://pkg.go.dev/github.com/spf13/pflag#FlagSet.MarkHidden , I'm not sure we need to show it. Let's clarify first that it doesn't violate GDPR. It is easier to un-hide it later than remove